scripts/deploy/cht-deploy

```diff
@@ -1,7 +1,33 @@
 #!/usr/bin/env node
 
-import install from './src/install.js';
+import { install } from './src/install.js';
 import fs from 'fs';
+import semver from 'semver';
+import path from 'path';
+
+import { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+function validateNodeVersion() {
+    const packageJsonPath = path.resolve(__dirname, 'package.json');
+    let packageJson;
```

Comment:
super minor, but we don't change this later, so we can `const` it up!

```suggestion
    const packageJson;
```

Reply:
Actually, we do. Check out the code block right below